src/bin/tikv-server.rs

```diff
@@ -233,6 +242,12 @@ fn run_raft_server(pd_client: RpcClient, cfg: &TiKvConfig, security_mgr: Arc<Sec
     ).unwrap_or_else(|e| fatal!("failed to start node: {:?}", e));
     initial_metric(&cfg.metric, Some(node.id()));
 
+    // Start auto gc
+    let auto_gc_cfg = AutoGCConfig::new(pd_client, region_info_accessor.clone(), node.id());
```

Comment:
Why use `AutoGCConfig`? Is `Auto` proper?

Reply:
Seems all these are contexts not configurations?

Reply:
TiKV does GC automatically without being requested, so I called it auto gc.
I was avoiding to use the phrase "distributed gc" in TiKV's code, because TiKV's code doesn't need to care how the GC process is distributed, but it simply does GC automatically by itself.
There's something named `GCWorkerContext` already...  I have no better idea about its naming.